doc/Build_on_docker.md

```diff
@@ -10,9 +10,68 @@ This on MacOS Catalina (10.15.3) and Mojave(10.14).
 - `sh docker_build.sh`
 - Somewhere in your host OS, run `socat TCP-LISTEN:6000,reuseaddr,fork UNIX-CLIENT:\"$DISPLAY\"`
 - Add your host machine IP address to xterm with `xhost` command.
-- Run docker with `sh quisp_docker.sh`
+- Run docker with `sh docker_run.sh`
 - Run `omnetpp` command on the docker container
 
+---
+
+## Easier way (Beta)
+
+This is just a beta version. If it's not working properly, please tell us as an issue.
+If you can't build with this process, please refer latter part.
+
+1. Open your terminal and run
+
+```zsh
+$ git clone https://github.com/sfc-aqua/quisp.git
+```
+
+You can download quisp from github.
+Enter quisp with `cd quisp`
+2. Run shell script
+
+```zsh
+$ sh docker_tools.sh
+```
+
+After you could successfully install related tools, please reboot your laptop. (Maybe just rebooting your terminal is enough.)
+
+3. Build docker container
+
+Open terminal and move to the `quisp/`, 
+
+```zsh
+$ sh docker_build.sh
+```
+
+You can see there is docker image called quisp with `docker images`.
```

Comment:
A docker image called quisp should have been created. You can check with the command `docker images`.